src/Gui/PythonWrapper.cpp

```diff
@@ -261,7 +261,9 @@ class WrapperManager : public QObject
         }
 
         auto& pylist = wrappers[obj];
-        if (std::find_if(pylist.cbegin(), pylist.cend(), [pyobj](const Py::Object& py) { return py.ptr() == pyobj; }) == pylist.end()) {
+        if (std::find_if(pylist.cbegin(), pylist.cend(), [pyobj](const Py::Object& py) {
+            return py.ptr() == pyobj;
+        }) == pylist.end()) {

```

Comment:
what's the correct way to format this?

Reply:
Not sure it matches our guidelines but I'd have done something like
```suggestion
        if (std::find_if(pylist.cbegin(), pylist.cend(),
                [pyobj](const Py::Object& py) {
                    return py.ptr() == pyobj;
                }) == pylist.end()) {
```

Reply:
Done, see [b3f5108](https://github.com/FreeCAD/FreeCAD/pull/7933/commits/b3f51082f945dbc40943d9e0e75b7a65e4d307e9)

Reply:
This commit doesn't exist probably because you rebased. ;) No problem.
Could you fix the typo in the last commit message ? :)
You can eventually squash last 2 commits too.